frontend/packages/core/src/NPS/emojiRatings.tsx

```diff
@@ -31,15 +31,15 @@ const EmojiRatings = ({ ratings = [], setRating }: EmojiRatingsProps) => {
   const [selectedRating, selectRating] = useState<Rating>(null);
 
   const StyledIconButton = styled(IconButton)<{
-    selected: boolean;
+    $selected: boolean;
```

Comment:
@jdslaugh is it ok to filter this out from the resulting element? I know there are tests that asserted this was set.

Reply:
Yeah its fine, it was a double check when verifying the opacity but I can just verify the styles themselves